src/main/java/uk/gov/hmcts/darts/audit/service/AudiServiceImpl.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.darts.audit.service;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.PersistenceContext;
+import jakarta.persistence.TypedQuery;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Predicate;
+import jakarta.persistence.criteria.Root;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.darts.audit.model.AuditSearchQuery;
+import uk.gov.hmcts.darts.common.entity.Audit;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+@Service
+@Slf4j
+public class AudiServiceImpl implements AuditService {
```

Comment:
Should be renamed to be `AuditServiceImpl`

Reply:
ACCEPT